aws/authn_task.json

```diff
@@ -0,0 +1,248 @@
+    "cpu": "256",
+    "memory": "512",
+    "containerDefinitions": [
+        {
+            "name": "sumologic-firelens",
+            "image": "public.ecr.aws/aws-observability/aws-for-fluent-bit:stable",
+            "cpu": 0,
+            "portMappings": [],
+            "essential": true,
+            "environment": [],
+            "mountPoints": [],
+            "volumesFrom": [],
+            "user": "0",
+            "firelensConfiguration": {
+                "type": "fluentbit",
+                "options": {
+                    "enable-ecs-log-metadata": "true"
+                }
+            }
+        },
+        {
+            "name": "newrelic-infra",
+            "image": "newrelic/nri-ecs:1.9.2",
+            "cpu": 0,
+            "portMappings": [],
+            "essential": true,
+            "secrets": [
+                {
+                    "name": "NRIA_LICENSE_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:monitoring/newrelic:api_key::"
+                }
+            ],
+            "environment": [
+                {
+                    "name": "NEW_RELIC_HOST",
+                    "value": "gov-collector.newrelic.com"
+                },
+                {
+                    "name": "NEW_RELIC_APP_NAME",
+                    "value": "{{program}}-{{environment}}-{{project}}-files"
+                },
+                {
+                    "name": "NRIA_IS_FORWARD_ONLY",
+                    "value": "true"
+                },
+                {
+                    "name": "NEW_RELIC_DISTRIBUTED_TRACING_ENABLED",
+                    "value": "true"
+                },
+                {
+                    "name": "NRIA_PASSTHROUGH_ENVIRONMENT",
+                    "value": "ECS_CONTAINER_METADATA_URI,ECS_CONTAINER_METADATA_URI_V4,FARGATE"
+                },
+                {
+                    "name": "FARGATE",
+                    "value": "true"
+                },
+                {
+                    "name": "NRIA_CUSTOM_ATTRIBUTES",
+                    "value": "{\"nrDeployMethod\":\"downloadPage\"}"
+                },
+                {
+                    "name": "NRIA_OVERRIDE_HOST_ROOT",
+                    "value": ""
+                }
+            ],
+            "mountPoints": [],
+            "volumesFrom": []
+        },
+        {
+            "name": "authn",
+            "image": "{{image_id}}",
+            "cpu": 0,
+            "portMappings": [
+                {
+                    "containerPort": 8082,
+                    "hostPort": 8082,
+                    "protocol": "tcp"
+                }
+            ],
+            "essential": true,
+            "entryPoint": [],
+            "secrets": [
+                {
+                    "name": "INDEXD_URL",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:{{account_id}}:secret:bento/{{project}}/{{environment}}:indexd_url::"
+                },
```

Comment:
I think Indexd is specific to the files service - this should probably be removed, Yizhen can tell us the correct list of environment variables to include